### ISSUE TITLE
ReVanced Stops Playing

### ISSUE THREAD

**Issue by NickDeBeenSAE** [ReVanced/revanced-patches#3113]
### Bug description

Spoof App Signature is not found in the settings to fix this.

Videos just stop playing in general.

Also, even though my phone is "adequate" for the time being, the app crashes quite a lot, and have had no way of fixing it.

Clearing cache clearly does nothing according to what I am reading, and neither does re-installing.

It's a bug, and needs to be fixed.

### Error logs

```shell
N/a*
```

### Solution

Remember to fix bugs before you bring out a new application.

### Additional context

I have nothing more to say.

The issues here are already in front of us, so, what is there more to say?

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by SodaWithoutSparkles** [comment]
We cant help you if you dont provide more information such as the version patched and list of patches used. 

We need to replicate the issue on our side before we can fix it, it's not like we have your phone or we can use magic `¯\_(ツ)_/¯`

**Comment by NickDeBeenSAE** [comment]
Well, the Discord chat has advised that ReVanced is currently broken for all users.

**Comment by NickDeBeenSAE** [comment]
> We cant help you if you dont provide more information such as the version patched and list of patches used.
> 
> We need to replicate the issue on our side before we can fix it, it's not like we have your phone or we can use magic `¯\_(ツ)_/¯`

Also, You'll have to stick with what I have advised as everything else has been covered in issues your squad has addressed already.

**Comment by NickDeBeenSAE** [comment]
Oh s***, I forgot to advise the version of YouTube I had, which I am now using Grayjay in the meantime:

19.0.43

**Comment by oSumAtrIX** [comment]
See https://github.com/ReVanced/revanced-discussions/discussions/1037#discussioncomment-9301571
